evalml/tests/model_understanding_tests/test_graphs.py

```diff
@@ -1028,3 +1032,73 @@ class NotTSPipeline:
     error_msg = "graph_prediction_vs_actual_over_time only supports time series regression pipelines! Received regression."
     with pytest.raises(ValueError, match=error_msg):
         graph_prediction_vs_actual_over_time(NotTSPipeline(), None, None, None)
+
+
+def test_get_tree_data(tree_estimators, logit_estimator):
```

Comment:
Suggestion on test formatting: maybe have a separate test to test non-tree estimators erroring out, and name that descriptively `test_get_tree_data_non_tree_errors` or something, and then have this test be the one that tests main functionality? :) Just so it's more clear what test is doing what!